### PR TITLE
Fixed #28303 -- Prevented localization of attribute values in the DTL attrs.html widget template.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/28303

### CODE REVIEW

**Comment by timgraham** [comment]
Maybe there's a more elegant solution -- I feel like we're playing whack-a-mole with numeric attributes. For example, the `step` attribute is also affected, even if a step > 999 is unlikely. `step` works with the following input types: number, range, date, datetime-local, month, time, and week. If we accept the solution in this PR, then I guess coercing of `step` could be done in `Widget.build_attrs()` to avoid repeating that in subclasses.

\cc @jdufresne @prestontimmons.

**Comment by claudep** [comment]
Or use `{% unlocalize %}` in `attrs,html`?

**Comment by prestontimmons** [comment]
+1 on Claude's idea.

**Comment by timgraham** [comment]
Sounds good; updated to that approach.

**Comment by jdufresne** [comment]
I believe this solution should make it easier to solve [ticket #28176](https://code.djangoproject.com/ticket/28176). If this lands, the option's input's value could be added to the `attrs` dict in `Wiget.create_option()`, then `value` could resume to be the actual value, not the coerced value.

**Comment by claudep** [comment]
By the way, what about including the localize/unlocalize tags in buit-in tags? It's unrelated, but typically in this situation, not having to call `{% load l10n %}` in a potentially frequently-called template include might give a small performance gain.

File: docs/releases/1.11.3.txt [hunk 1]
```diff
@@ -40,3 +40,7 @@ Bugfixes
   context. It's now an empty string (as it is for the original function-based
   ``login()`` view) if the corresponding parameter isn't sent in a request (in
   particular, when the login page is accessed directly) (:ticket:`28229`).
+
+* Disabled localization in the ``django/forms/widgets/attrs.html`` template so
+  that the values of the numeric attributes (e.g. ``max`` and ``min``) of
+  ``NumberInput`` appear correctly (:ticket:`28075`).
```
Comment (claudep) [inline docs/releases/1.11.3.txt]: Isn't it `28303`?

**Comment by claudep** [comment]
I think that 1442e2998 could then be partially reverted (not the test part).

**Comment by timgraham** [comment]
I haven't seen any profiling about the cost of `{% load ... %}` but that could be something to look at.

1442e299839febf9599e98dff2e6bb6da24df0ac can't be reverted because the `value` isn't in rendered in the `attrs.html` template. I'm trying to develop a solution and will send a separate PR.

**Comment by timgraham** [comment]
I noticed that `|localize` (which does `str(value)`) has a different behavior from `{% localize off %}`. In the latter case, datetimes/times are still formatted, e.g. 00:00:00 becomes midnight. To avoid the need for `{% load l10n %}` maybe `|stringformat:'s'` is a preferred solution.

**Pull request merged**